engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java

```diff
@@ -145,4 +160,25 @@ public void migrateProcessMessageSubscriptionSentTime(
           processSubscriptionSentTimeColumnFamily.delete(key);
         });
   }
+
+  @Override
+  public void migrateTemporaryVariables(
+      final MutableEventScopeInstanceState eventScopeInstanceState) {
+    temporaryVariableColumnFamily.forEach(
+        (key, value) -> {
+          // Event scope key can be a static value. Together with the
+          final long eventScopeKey = -1L;
```

Comment:
:x: We should rename the variable to `eventKey` because it is used as the argument for the event key.